product_docs/docs/biganimal/release/free_trial/detail/experiment/index.mdx

```diff
@@ -24,3 +25,7 @@ Well, if you want to configure a new cluster, you can just delete the currently
 4. Reload the page.
 
 Now the **Create New Cluster** button is enabled again, and you can [create a new cluster](../create_a_cluster) or [restore a previously deleted one](backup_and_restore).
+
+## Linking to an Oracle database
```

Comment:
Check with SME if this is relevant in the Free Trial section, because Private clusters aren't allowed in a free trial.

Reply:
I'm going to remove the section. You can add it back in later if you learn it is relevant.